src/catalogue/items/catalogueItemsDialog.component.tsx

```diff
@@ -526,6 +526,21 @@ function CatalogueItemsDialog(props: CatalogueItemsDialogProps) {
       .concat(sortDataList(classifiedManufacturers, 'name').reverse());
   };
 
+  const errorMessage = ( message: string | undefined ) => {
+    if (message?.includes('child elements')) {
+      // find the name of the manufacturer, so it can be used in the error message
+      const manufacturerName = manufacturerList?.find(
+        (manufacturer) => manufacturer.id === selectedCatalogueItem?.manufacturer_id
+      ) || null;
+      // add the manufacturer name into the error message
+      const childElementsMessage = "Unable to update catalogue item properties and manufacturer ("
+        + manufacturerName?.name + "), as the catalogue item has associated items.";
+      return (childElementsMessage);
+    } else {
+      return (message);
+    };
+  };
+
```

Comment:
The error message for child elements should be set on line 390 in the `setErrorPropertiesStep ` not on line 1020. You shouldn't need to use a function either.